pkg/devspace/configure/deployment.go

```diff
@@ -141,8 +141,18 @@ func (m *manager) AddHelmDeployment(deploymentName string) error {
 				localChartPathRel = localChartPath
 			}
 
-			stat, err := os.Stat(path.Join(localChartPathRel, "Chart.yaml"))
-			if err != nil || stat.IsDir() {
+			pathStat, err := os.Stat(localChartPath)
+			if err != nil {
+				return err
+			}
+
+			if !pathStat.IsDir() {
+				m.log.WriteString(logrus.InfoLevel, "\n")
+				m.log.Errorf("Local path `%s` is not a Helm chart (path is not a directory)", localChartPathRel)
+				continue
+			}
+
+			if _, err := os.Stat(path.Join(localChartPathRel, "Chart.yaml")); errors.Is(err, os.ErrNotExist) {
```

Comment:
Would this hide other potential errors? Perhaps something like the following would prevent that:
```golang
if _, err := os.Stat(path.Join(localChartPathRel, "Chart.yaml")); err != nil {
  m.log.WriteString(logrus.InfoLevel, "\n")
  if errors.Is(err, os.ErrNotExist) {
    m.log.Errorf("Local path `%s` is not a Helm chart (Chart.yaml missing)", localChartPathRel)
    continue
  } else {
    m.log.Errorf("Encountered unexpected error checking local path `%s`: %s", localChartPathRel, err.Error())
    continue
  }
}
```

Reply:
Makes sense. Definitely a good idea to include handling unknown errors too. Changed.